components/frontend/src/source/SourceEntityDetails.js

```diff
@@ -27,12 +27,11 @@ entityStatusOption.propTypes = {
 }
 
 function entityStatusOptions(entityType, report) {
-    const desiredResponseTimes = report?.desired_response_times ?? {}
     const unconfirmedSubheader = `This ${entityType} should be reviewed to decide what to do with it.`
-    const confirmedSubheader = `This ${entityType} has been reviewed and should be dealt with within ${desiredResponseTimes["confirmed"]} days.`
-    const fixedSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["fixed"]} days because it has been fixed or will be fixed shortly.`
-    const falsePositiveSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["false_positive"]} days because it's been incorrectly identified as ${entityType}.`
-    const wontFixSubheader = `Ignore this ${entityType} for ${desiredResponseTimes["wont_fix"]} days because it will not be fixed.`
+    const confirmedSubheader = `This ${entityType} has been reviewed and should be dealt with within ${getDesiredResponseTime(report, "confirmed")} days.`
+    const fixedSubheader = `Ignore this ${entityType} for ${getDesiredResponseTime(report, "fixed")} days because it has been fixed or will be fixed shortly.`
```

Comment:
I'm not sure that "shortly" is correct, perhaps "promptly" or "imminently" is more suitable?

Reply:
I think shortly is fine, see second example here: https://dictionary.cambridge.org/dictionary/english/shortly